client/components/quizSingle/styles.js

```diff
@@ -171,4 +171,10 @@ export const ContentWrapper = styled.div`
   max-width: calc(100% - 70px);
   margin: 0 auto;
   color: ${props => props.theme.colors.light};
-`;
\ No newline at end of file
+`;
+
+export const SubmitBtnStyled = styled(PrimaryButton)`
```

Comment:
Same question as in your other PR 😅. Since this is an exact copy of the button just above in this file, would you change it to a more universal name and use it for all the buttons in the quiz?

Reply:
Replaced with generic button.